src/Compilers/CSharp/Test/Semantic/Semantics/StaticNullChecking_FlowAnalysis.cs

```diff
@@ -462,12 +471,12 @@ class C
 {
     static void F(bool b, A<object> x, B1 y)
     {
-        (b ? x : y)/*T:A<object!>!*/.F.ToString();
+        (b ? x : y)/*T:A<object>!*/.F.ToString();
```

Comment:
>object [](start = 25, length = 6)

Just to confirm: `object` (without `!`) means oblivious in `VerifyTypes` comments, right?
Could you take notes somewhere (speclet would be best) to document the behavior (failed inference of nullability results in oblivious type).

Reply:
Yes `object` without `!` is oblivious. There is no warning on the use of the nested type from the field or property: `(b ? x : y).F`.

---
In reply to: [189331979](https://github.com/dotnet/roslyn/pull/26356#discussion_r189331979) [](ancestors = 189331979)